packages/perseus-editor/src/widgets/dropdown-editor.tsx

```diff
@@ -147,11 +147,11 @@ class DropdownEditor extends React.Component<Props> {
                             exercise is accessible. For more information on
                             writing accessible labels, please see{" "}
                             <a
-                                href="https://www.w3.org/WAI/tips/designing/#ensure-that-form-elements-include-clearly-associated-labels"
+                                href="https://khanacademy.atlassian.net/wiki/spaces/A11Y/pages/3967615350/General+Guidance+for+Labeling+Dropdowns"
                                 target="_blank"
                                 rel="noreferrer"
                             >
-                                this article.
+                                this internal article.
```

Comment:
My one question is: can we link to an internal article in this project? Or am I not adequately considering external audiences since this is an OS project?

Reply:
I personally think it would be preferable to avoid linking to internal links from the UI.

Reply:
Thanks for the response! How do we give relevant advice to authors, in that case? Publish something external?